.helm/ecamp3/values.yaml

```diff
@@ -91,10 +91,25 @@ browserless:
   image:
     repository: "docker.io/browserless/chrome"
     pullPolicy: IfNotPresent
-    tag: "1.53-puppeteer-13.6.0"
+    tag: "1.56.0-puppeteer-18.0.5"
```

Comment:
Note to self, we still need to link the puppeteer version here using a renovate regex.

Reply:
#3120